### DISCUSSION TITLE
How to build a site without `docs` in `package.json` like `vuejs/docs`

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [vuejs/vitepress#1427] [Q&A] [answered]
I'm a beginner. What I want is not to build "a project docs", just a site like blog. so I want a directory structure like [vuejs/docs](https://github.com/vuejs/docs). I found the [`package.json`](https://github.com/vuejs/docs/blob/main/package.json)'s `scripts` are `vite serve` instead of `vite serve src` like [vitepress doc](https://vitepress.vuejs.org/guide/getting-started#step-3-boot-up-dev-environment).

**Comment by brc-dd** [comment, marked as answer]
That docs directory isn't required. Simply running `vitepress` for dev and `vitepress build` will work too.

**Reply by awxiaoxian2020** [reply]
I will get a 404 page .
![1](https://user-images.githubusercontent.com/69554233/193456389-d4871b1e-e537-4d23-ab4f-f01f4c88b998.png)

**Reply by awxiaoxian2020** [reply]
What should I do? @brc-dd

**Reply by brc-dd** [reply]
Your code is inside src. So your commands should be `vitepress dev src` and `vitepress build src`.

**Reply by awxiaoxian2020** [reply]
But the https://github.com/vuejs/docs/blob/main/package.json has no `vitepress dev src`, Why? @brc-dd

**Reply by brc-dd** [reply]
Because they have `srcDir` set to `'src'` in their `.vitepress/config`.

Also, in case of `vitepress dev src`, you'll need to place your `.vitepress` directory inside `src`. `vuejs/docs` have that at project root.

**Reply by awxiaoxian2020** [reply]
Thanks for your reply! @brc-dd  I think the `srcDir` can be documented(I haven't found it in docs now)

**Reply by brc-dd** [reply]
> I think the srcDir can be documented(I haven't found it in docs now)

PRs are welcome. Ideally it should be mentioned here: https://vitepress.vuejs.org/config/app-configs
